source/org/zfin/publication/repository/HibernatePublicationRepository.java

```diff
@@ -1,2584 +1,2596 @@
-        cr.where(criteriaBuilder.equal(root.get("publication").get("zdbID"), pubID));
-        currentSession().createQuery(cr).executeUpdate();
-    }
-
-    @Override
-    public CorrespondenceResolutionType getCorrespondenceResolutionTypeByID(long id) {
-        return currentSession().get(CorrespondenceResolutionType.class, id);
-    }
-
-    @Override
-    public void insertCorrespondenceResolution(CorrespondenceResolution correspondenceResolution) {
-        currentSession().save(correspondenceResolution);
-    }
-
-    @Override
-    public Map<Publication, List<PublicationDbXref>> getAllDataSetsPublication() {
-        String hql = " select p from Publication p " +
-                     " where p.dbXrefs is not empty" +
-                     "";
-        Query<Publication> query = HibernateUtil.currentSession().createQuery(hql, Publication.class);
-        Map<Publication, List<PublicationDbXref>> returnMap = new HashMap<>();
-        query.list().forEach(publication -> {
-            List<PublicationDbXref> xrefs = returnMap.computeIfAbsent(publication, k -> new ArrayList<>());
-            xrefs.addAll(publication.getDbXrefs());
-        });
-        return returnMap;
-    }
-
-    private Map<Publication, List<STRTargetRow>> strMaps;
-
-    @Override
-    public Map<Publication, List<STRTargetRow>> getAllAttributedSTRs(Pagination pagination) {
-        if (strMaps != null)
-            return strMaps;
-        Session session = HibernateUtil.currentSession();
-        String hql = """
-            select marker, attr.publication from SequenceTargetingReagent marker, PublicationAttribution attr
-            where attr.dataZdbID = marker.zdbID
-            and marker.markerType.name in (:markerTypes)
-            """;
-        if (pagination.getFieldFilter(FieldFilter.STR_NAME) != null) {
-            hql += " AND marker.abbreviation like '%" + pagination.getFieldFilter(FieldFilter.STR_NAME) + "%'";
-        }
-        if (pagination.getFieldFilter(FieldFilter.STR_TYPE) != null) {
-            hql += " AND marker.zdbID like '%" + pagination.getFieldFilter(FieldFilter.STR_TYPE) + "%'";
-        }
-        Query<Tuple> query = session.createQuery(hql, Tuple.class);
-        query.setParameterList("markerTypes", List.of(Marker.Type.MRPHLNO.name(), Marker.Type.CRISPR.name(), Marker.Type.TALEN.name()));
-        Map<Publication, List<STRTargetRow>> map = new HashMap<>();
-        query.list().forEach(tuple -> {
-            SequenceTargetingReagent str = (SequenceTargetingReagent) tuple.get(0);
-            for (Marker target : str.getTargetGenes()) {
-                STRTargetRow row = new STRTargetRow(str, target);
-                List<STRTargetRow> rows = map.computeIfAbsent((Publication) tuple.get(1), k -> new ArrayList<>());
-                rows.add(row);
-            }
-        });
-        strMaps = map;
-        return strMaps;
-    }
-
-}
+package org.zfin.publication.repository;
```

Comment:
What changed in this file? It looks like all lines have changed. Was it a line ending?

Reply:
@cmpich That's weird. It looks like git changed the line endings or something. I just redid the changes using `git add --patch` and it seems to have fixed the diff view.